web/src/services/circuitbreaker.ts

```diff
@@ -0,0 +1,31 @@
+import axios from 'axios';
+
+const circuitbreakerApi = axios.create({
+  baseURL: '/api',
+});
+
+export const getInfo = async () =>
+  (await circuitbreakerApi.get<Info>('/info')).data;
+
+export const getLimits = async () =>
+  (await circuitbreakerApi.get<Limits>('/limits')).data;
+
+interface UpdateLimitsParams {
+  limits: {
+    [key: string]: Limit;
+  };
+}
+export const updateLimits = async (params: UpdateLimitsParams) =>
+  (await circuitbreakerApi.post<{}>('/updatelimits', params)).data;
```

Comment:
Created #62

Reply:
Also in the console: `Unchecked runtime.lastError: A listener indicated an asynchronous response by returning true, but the message channel closed before a response was received` - related?

Reply:
If a clean error message is more than a one liner, maybe a simple standard popup can be added in this PR so that there is at least something happening?